site/app/controllers/grading/ElectronicGraderController.php

```diff
@@ -311,7 +311,7 @@ public function showStatus($gradeable_id) {
         $this->core->getOutput()->addBreadcrumb("{$gradeable->getTitle()} Grading", $gradeableUrl);
 
         $isPeerGradeable = false;
-        if ($gradeable->isPeerGrading() && ($this->core->getUser()->getGroup() == User::GROUP_INSTRUCTOR)) {
+        if ($gradeable->isPeerGrading() && ($this->core->getUser()->getGroup() == User::GROUP_INSTRUCTOR || $this->core->getUser()->getGroup() == User::GROUP_FULL_ACCESS_GRADER || $this->core->getUser()->getGroup() == User::GROUP_LIMITED_ACCESS_GRADER)) {
```

Comment:
You can just say $this->core->getUser()->getGroup() < User:GROUP_STUDENT

Reply:
done!